client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -1392,3 +1400,20 @@ export const openDetailsDialog = ( orderId, siteId, labelId ) => {
 export const closeDetailsDialog = ( orderId, siteId ) => {
 	return { type: WOOCOMMERCE_SERVICES_SHIPPING_LABEL_CLOSE_DETAILS_DIALOG, orderId, siteId };
 };
+
+export const setHazmatType = ( { hazmatType, siteId, orderId } ) => {
```

Comment:
Curious why the param here is an object but the rest of the actions aren't?

Reply:
Adjusted in https://github.com/Automattic/woocommerce-services/pull/2664/commits/d95c7e20b0fe3c5768673e375c7ca68464d88948